src/scheduled-tasks/messages/dailyActivism.ts

```diff
@@ -0,0 +1,226 @@
+    'from animal ag to plant based farming.',
+    
+    '**In Germany 🇩🇪?** Get connected with [V-Party3](<https://v-partei.de/>) for legislation activism!\n'+
+    '*Die V-Partei ist eine deutsche Partei, die der Tierproduktindustrie den Kampf angesagt hat,'+
+    'mit Verboten jeglicher tierischen Produkten, Tierversuchen und Zurschaustellung in Zoo und Circus.'+
+    'Zusätzlich setzen sie sich für ernstzunehmende ethische und ernährungstechnische Bildung an Schulen'+
+    'und den Schutz von Tierrechtsaktivisten ein und fördern bezahlbare Nahrungsmittel aus'+
+    'solidarischer Landwirtschaft mit ökologischen Alternativen zu Pestiziden.*',
+    
+    '**Need a cheatsheet for responding to justifications to harm and exploit animals?**'+
+    'Bookmark [Vegan Sidekick](<https://www.godfist.com/vegansidekick/guide.php>).'+
+    'A comprehensive list of the known excuses and the responses for them.'+
+    'It will also link to the common comebacks after responding to certain excuses too!',
+    
+    '**Looking for a streaming service of Plant-Based News & Entertainment Network for FREE?**'+
+    'Download [UnchainedTV](<https://unchainedtv.com/>) on your phone via the APP store'+
+    'or on your TV via your Amazon Fire Stick, AppleTV device or Roku device.',
+    
+    '**Need some food content creator recommendations?**\n'+
+    '🔹[Nora Cooks](<https://www.noracooks.com/>) - Recipes that are easy to make and even easier to eat\n'+
+    '🔹[Forks Over Knives](<https://www.forksoverknives.com/recipes/>) - Healthy whole food plant-based recipes\n'+
+    '🔹[Rainbow Plant Life](<https://rainbowplantlife.com/>) - For the home cook looking to wow their friends\n'+
+    '🔹[Cheap Lazy Vegan](<https://thecheaplazyvegan.com/blog/>) - Easy and affordable vegan meal ideas\n'+
+    '🔹[The Foodie Takes Flight](<https://thefoodietakesflight.com/>) - Asian-inspired recipes\n'+
+    '🔹[Vegan Richa](<https://www.veganricha.com/recipes/>) - Indian-inspired recipes\n'+
+    '🔹[Eat Figs Not Pigs](<https://www.eatfigsnotpigs.com/>) - Fusion comfort foods\n'+
+    '🔹[Thee Burger Dude](<https://theeburgerdude.com/>) - Popular fast food recipes veganized',
+    
+    '**Need a comprehensive source on vegan nutrition?**'+
+    '[Vegan Health](<https://veganhealth.org/>) is a website with sources and studies'+
+    'by registered dieticians on evidence-based nutrient recommendations.',
+    
+    '**Looking for activism opportunities and events near you?**'+
+    'Try [Animal Rights Calender](<https://animalrightscalendar.com/>)!'+
+    'Not finding an organization or event? Contact email: *person@animalrightscalendar.com*'
+  ];
+
+  public async run() {
+    // Get the total messages sent in non-vegan general since last message
+    const redisKey = 'verifyReminderMessageCounter';
+    
+    const messageCount = await this.container.redis.get(redisKey);
+    
+    // Do not send if messages count is less than 100
+    if (!messageCount || +messageCount < 100) return;
```

Comment:
It might be better if the message count something like 10 or 20? Mostly due to the activity.